packages/web/src/pages/sign-up-page/pages/SelectGenrePage.tsx

```diff
@@ -52,40 +53,76 @@ export const SelectGenrePage = () => {
     [dispatch, navigate]
   )
 
+  const { isMobile } = useMedia()
+
   return (
-    <div>
-      <div>
-        <p>{displayName}</p>
-        <p>{handle}</p>
-      </div>
-      <h1>{messages.header}</h1>
-      <p>{messages.description}</p>
+    <Flex direction='column' gap={isMobile ? '2xl' : '4xl'}>
+      <AccountHeader />
+      <Flex direction='column' gap={isMobile ? 'xl' : '2xl'}>
+        <Flex direction='column' gap={isMobile ? 's' : 'l'} ph='l'>
+          <Text
+            variant='heading'
+            size={isMobile ? 'm' : 'l'}
+            color='heading'
```

Comment:
does heading not default to heading color? If not, should it?

Reply:
great question. it does not currently. @szonana ?

Reply:
It absolutely can in our signup flow, but i don't think it should for Harmony